distributions/openhab/src/main/resources/userdata/etc/custom.properties

```
@@ -5,6 +5,8 @@ karaf.lock.dir=${karaf.data}/tmp
 karaf.shutdown.port.file=${karaf.data}/tmp/port
 karaf.pid.file=${karaf.data}/tmp/karaf.pid
 
+org.osgi.framework.system.packages.extra=${org.osgi.framework.system.packages.extra},sun.security.pkcs11
```

What happens if you use 
```suggestion
org.osgi.framework.system.packages.extra=sun.security.pkcs11
```

---

It overwrites the existing value (multiple packages) set in the config.properties. Karaf won’t start.